src/Typeclasses/Classes/Bounded.elm

```diff
@@ -0,0 +1,123 @@
+module Typeclasses.Classes.Bounded exposing
+    ( Bounded
+    , interval
+    , int, char, bool, order, unit
+    , tuple2, tuple3
+    )
+
+{-| Bounded typeclass definition and its instances for basic types.
+
+
+# Definition
+
+@docs Bounded
+
+
+# Construction utilities
+
+@docs interval
+
+
+# Instances
+
+@docs int, char, bool, order, unit
+
+
+# Composites
+
+@docs tuple2, tuple3
+
+-}
+
+
+{-| Explicit typeclass which implements a bounded type `a`.
+-}
+type alias Bounded a =
+    { minBound : a
```

Comment:
The `...Bound` suffix is redundant. Let's go with just `min` and `max`.

Reply:
Done.